src/base/kaldi-error.h

```diff
@@ -63,60 +66,51 @@ inline int32 GetVerboseLevel() { return g_kaldi_verbose_level; }
 /// automatically from ParseOptions.
 inline void SetVerboseLevel(int32 i) { g_kaldi_verbose_level = i; }
 
-// Class KaldiLogMessage is invoked from the  KALDI_WARN, KALDI_VLOG and
-// KALDI_LOG macros. It prints the message to stderr.  Note: we avoid
-// using cerr, due to problems with thread safety.  fprintf is guaranteed
-// thread-safe.
-
-// class KaldiWarnMessage is invoked from the KALDI_WARN macro.
-class KaldiWarnMessage {
- public:
-  inline std::ostream &stream() { return ss_; }
-  KaldiWarnMessage(const char *func, const char *file, int32 line);
-  ~KaldiWarnMessage();
- private:
-  std::ostringstream ss_;
+/// Log message severity and source location info.
+struct LogMessageEnvelope {
+  // Positive number cast to LogLevel is for verbose logging.
```

Comment:
I removed this sloppy comment and updated the following one (LogLevel was the née of the enum Severity), plus couple other minor changes: https://github.com/kaldi-asr/kaldi/pull/642/commits/408c22d356e21c805be9856f6d4291f9b0eab3de

Squash?